localisation/000_Other_title_names.csv

```diff
@@ -396,8 +396,8 @@ e_wastewander;Wastewander;;;;;;;;;;;;;x
 e_wastewander_adj;Wastewander;;;;;;;;;;;;;x
 d_darkmoon_island;Darkmoon Island;Darkmoon Island;Dunkelmond Insel;;Darkmoon Island;;;;;;;;;x
 d_darkmoon_island_adj;Darkmoon;;dunkelmondisch;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+PROV94;Jaguero;;;;;;;;;;;;;x
+PROV2220;Bloodsail;;;;;;;;;;;;;x
```

Comment:
County adjectives not changed.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/81499180-97787080-92da-11ea-98be-2a02196ca176.png)

</details>